modules/crud/sql/transaction.go

```diff
@@ -0,0 +1,65 @@
+package sql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/spaceuptech/space-cloud/model"
+)
+
+func (s *SQL) Transaction(ctx context.Context, project string, reqs *model.TransactionRequest) error {
+
+	var opts *sql.TxOptions
```

Comment:
Is opts supposed to be empty?

Reply:
Opts has two fields IsolationLevel and one boolean ReadOnly 
What value should i give ?

Reply:
Turns out we can pass `nil` instead of options to use the defaults. Can we explicitly put nil and remove that opts variable. Probably add a todo comment saying opts need to be created?

Reply:
Yes nil can be passed . Added todo